src/app/core/services/issue-comment.service.ts

```diff
@@ -45,6 +47,27 @@ export class IssueCommentService {
     );
   }
 
+  createGithubTesterResponse(teamResponse: string, testerResponses: TesterResponse[]): string {
+    return `# Team\'s Response\n${teamResponse}\n ` +
+          `# Items for the Tester to Verify\n${this.getTesterResponsesString(testerResponses)}`;
+  }
+
+  createGithubTutorResponse(issueDisputes: IssueDispute[]): string {
+    let tutorResponseString = '# Tutor Moderation\n\n';
+    for (const issueDispute of issueDisputes) {
+      tutorResponseString += issueDispute.toTutorResponseString();
+    }
+    return tutorResponseString;
+  }
+
+  private getTesterResponsesString(testerResponses: TesterResponse[]): string {
+    let testerResponsesString = '';
+    for (const testerResponse of testerResponses) {
```

Comment:
@KevinCJH 

Do you think it will be cleaner if we did it like this?
```typescript
testerResponses.forEach((testerResponse: TesterResponse) => {
  testerResponsesString.concat(testerResponse.toString());
}
```

Instead of the for-loop?